keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -523,11 +616,11 @@ The previous PodAffinity/PodAntiAffinity behavior will not be broken. Users can
 their Pod specs as it is.
 
 To use this enhancement, users need to enable the feature gate (during this feature is in the alpha.),
-and add `MatchLabelKeys` on their PodAffinity/PodAntiAffinity.
+and add `MatchLabelSelectors` on their PodAffinity/PodAntiAffinity.
 
 **Downgrade**
 
-kube-apiserver will ignore `MatchLabelKeys` in PodAffinity/PodAntiAffinity,
+kube-apiserver will ignore `MatchLabelSelectors` in PodAffinity/PodAntiAffinity,
```

Comment:
Actually - it will ignore (drop) setting this field for newly added objects.
But existing objects will continue to exist.

So I'm assuming that kube-scheduler will also be performing the actual computations under feature-gate, right?

Reply:
This is also touching on
"How can this feature be enabled / disabled in a live cluster?" in PRR.

Reply:
> existing objects will continue to exist.

Ah, yes correct. This PR changes MatchLabelSelectors to be handled in kube-apiserver when Pod is created. 
So, existing objects will continue to have the labelselector made by MatchLabelSelectors.

What is the ideal action here? OK to leave the existing objects when turning off the feature gate, or we somehow need to ignore the labelselector made by MatchLabelSelectors (gonna be tricky though..).